docs/references/rest-apis/rest-system-api.md

```diff
@@ -0,0 +1,389 @@
+!!! note
+    This API can also be accessed via the RequestHandler with app-id: `SYS-V1`.
+
+The `SystemService` APIs return properties that are divided into 3 categories:
+
+- **framework** properties,
+- **extended** properties, and
+- **kura** properties.
+
```

Comment:
Can you add a description of what kind of properties are these?

Reply:
Sure